src/components/OrderForm/OrderForm.helpers.js

```diff
@@ -44,6 +54,15 @@ const marketToQuoteBase = (market) => ({
   BASE: market.base,
 })
 
+// 'tBTC:USD' -> { QUOTE: 'USD', BASE: 'BTC' }
+const symbolToQuoteBase = (symbol) => {
+  const market = _split(symbol, ':')
+  return {
+    BASE: _replace(market[0], /^t/, ''),
+    QUOTE: market[1],
+  }
+}
+
```

Comment:
I think this conversion not always valid for all pairs, i.e. IOTA/BTC pair it's symbols is sent as tIOTBTC in open algos state
could you ask backend team to help uniform this naming, or if possible send in format similar to wsID sent in `markets` event, so we could retrieve base quote from markets redux state

Reply:
ok, seems this logic should work mostly as I can not find any pair "having more than 7 characters and without : " 🤞